### PR TITLE
Speed up `docker compose down` for postgREST

### PR DESCRIPTION
**Pull request by zachblume**
## What kind of change does this PR introduce?

Speeds up `docker compose down` for postgREST from 10 seconds to 0.3 seconds
- By making the command explicit for postgREST in the docker compose config file.

## What is the current behavior?

`docker compose down` with the current config takes 10s, because postgREST doesn't have an explicit configuration which results in the default docker timeout of 10s being used after SIGTERM

https://docs.docker.com/compose/faq/#why-do-my-services-take-10-seconds-to-recreate-or-stop

## What is the new behavior?

`docker compose down` takes ~0.3s

## Additional context

Bringing info over here from postgREST's repo

https://github.com/PostgREST/postgrest/issues/2851

### CODE REVIEW

**Review by soedirgo — APPROVED** [review]

**Comment by soedirgo** [comment]
Sick!! This should be in the PostgREST image 😄

**Comment by Lan-Hekary** [comment]
Hello @zachblume,
I think it should be a single line. Like this
```
    command: "postgrest"
```
instead of
```
    command:
      command: "postgrest"
```
other than that. that's an awesome fix.
Thank You!

**Comment by zachblume** [comment]
On ARM containers, it needs to be ["postgrest"] apparently. See https://github.com/PostgREST/postgrest/pull/2852

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -127,6 +127,10 @@ services:
       PGRST_DB_ANON_ROLE: anon
       PGRST_JWT_SECRET: ${JWT_SECRET}
       PGRST_DB_USE_LEGACY_GUCS: "false"
+    # The following explicit command quickens the `docker compose down` command from 10s to 0.3s:
+    # https://github.com/PostgREST/postgrest/issues/2851
+    command:
+      command: "postgrest"
```
Comment (zachblume) [inline docker/docker-compose.yml:133]:
```suggestion
    command: "postgrest"
```

**Comment by zachblume** [comment]
Oops, deleted my previous comment because I misunderstood.

Implemented your changes, and looks like postgREST team is going to land a permanent fix in their Dockerfile

**Comment by Lan-Hekary** [comment]
Thanks @zachblume, 
I tested it, it checks out.

Awesome news!
The right thing to do is to remove this command once they implement the fix in the main image and once Supabase moves to the new image.
Right now Supabase uses v10.

**Comment by zachblume** [comment]
Do you all have a sense of when postgREST 11 might be adopted by supabase hosted? I picked this up because I have been migrating to self-hosting in order to mix in postgREST 11 to the docker compose etc etc because there's some new features they've shipped (re: relational embedding)? I would love to not do this and keep using hosted

**Comment by steve-chavez** [comment]
@zachblume PostgREST v11.1 was deployed this week. New supabase projects should have it. There will be an announcement soon.

**Comment by zachblume** [comment]
Pausing and restoring my project brought it up to speed -- thank you all, you all are champs! May continue playing around with self-hosting to see about instrumenting things more but it's really great to know you all roll things out that fast.

**Pull request merged**